src/home-page/HomePage.tsx

```diff
@@ -4,13 +4,28 @@ import { MainContainer } from '../layout/components/MainContainer';
 import { Header } from './components/Header';
 import { Description } from './components/Description';
 import { GridContainer } from './components/GridContainer';
+import { CategoryCard } from './components/CategoryCard';
+import { cards } from './data/data';
 
-export const HomePage: React.FC<{}> = () => (
-  <MainContainer>
-    <Header />
-    <Description />
-    <GridContainer>
-      {/* Fill with actual content */}
-    </GridContainer>
-  </MainContainer>
-);
+export const HomePage: React.FC<{}> = () => {
+  return (
+    <MainContainer>
+      <Header />
+      <Description />
+      <GridContainer>
+        {cards.map(({ title, description, color, icon }) => {
```

Comment:
We can add some comment here to remembered what is going on in the future. Becouse I think name "cards" is not self-documenting. Also we can think about changing name cards for something more self-explanatory eg. itProfesionsCards to make it more clear.

Reply:
I changed filename for ITProfessions as @vilanelle suggested above.